app/src/main/java/org/thoughtcrime/securesms/notifications/BackgroundPollWorker.kt

```diff
@@ -49,41 +81,89 @@ class BackgroundPollWorker(val context: Context, params: WorkerParameters) : Wor
             return Result.failure()
         }
 
+        // If this is a scheduled run and it is happening before the initial scheduled time (as
```

Comment:
Very helpful comment here, was wondering what the inital time stuff was doing :smile:

Reply:
Haha yea, it was causing issues when debugging calls because one of the open group requests was returning a `400` which was resulting in the poller retrying so it wasn't actually processing the call messages - once I'd plugged in a solution for that case I realised that some times it was timing out because it was polling everything concurrently which made the DM poll too slow